src/commands/distribute/groups/update.ts

```diff
@@ -91,16 +97,27 @@ export default class UpdateDistributionGroupCommand extends AppCommand {
       addedTestersEmails = [];
     }
 
+    if (deletedTestersEmails.length !== testersToDeleteEmails.length || addedTestersEmails.length !== testersToAddEmails.length) {
+      out.text("Updating the list of testers was partially successful");
+    }
+
     let currentGroupName: string;
+    const options: { name?: string; isPublic?: boolean } = {};
     if (!_.isNil(this.newDistributionGroupName)) {
       debug("Renaming the distribution group");
-      currentGroupName = await this.renameDistributionGroup(client, app, this.newDistributionGroupName);
+      options.name = this.newDistributionGroupName;
+      currentGroupName = this.newDistributionGroupName;
     } else {
       currentGroupName = this.distributionGroup;
     }
 
-    if (deletedTestersEmails.length !== testersToDeleteEmails.length || addedTestersEmails.length !== testersToAddEmails.length) {
-      out.text("Updating the list of testers was partially successful");
+    if (!_.isNil(this.isPublic)) {
+      debug("Setting distribution group public status to " + this.isPublic);
+      options.isPublic = JSON.parse(this.isPublic);
```

Comment:
How does it work when `isPublic` is unexpected value? I tested in the console with "TRUE" and it does not work.